### PR TITLE
#ARRUS-202, ARRUS-203: fix issue with TGC in Matlab, enable the RF display.

### PR DESCRIPTION
**Pull request by PiotrKarwat**
ARRUS-202:
- TGC range is dependent on LNA and PGA values.
- TGC delay and sampling are verified and corrected.

ARRUS-203:
- RF display is now available in Matlab using plotRawRf or imageRawRf methods of the Us4R class. 
- The maximum amplitude of the undistorted* signal can be optionally marked.
* distortions due to the AFE input voltage exceeding the allowed range (nonlinear behaviour observed outside this range)

### CODE REVIEW

**Comment by pjarosik** [comment]
@zkLog please revie

File: api/matlab/examples/Us4R_control_rawRfPwi.m [hunk 1]
```diff
@@ -0,0 +1,64 @@
+% Example script for b-mode imaging using plane waves and angular scanning
```
Comment (zkLog) [inline api/matlab/examples/Us4R_control_rawRfPwi.m:1]: This is not script for b-mode imaging.

**Review by zkLog — CHANGES_REQUESTED** [review]
(for future versions):
1. Consider adding the label with information which lines are displayed.
2. Consider adding the option for plot orientation selection, to have possibility to plot line horizontally. 

For approving this version - correct the docstring in us4R_control_rawRfPwi.m script.

File: api/matlab/examples/Us4R_control_rawRfPwi.m (hunk 1 above)
Comment (PiotrKarwat) [reply api/matlab/examples/Us4R_control_rawRfPwi.m]: docstring corrected

**Comment by zkLog** [comment]
Approved.

**Pull request merged**